Src/LinearSolvers/MLMG/AMReX_MLMG.H

```diff
@@ -374,6 +374,9 @@ MLMGT<MF>::solve (const Vector<AMF*>& a_sol, const Vector<AMF const*>& a_rhs,
 {
     BL_PROFILE("MLMG::solve()");
 
+    [[maybe_unused]] Gpu::SingleStreamRegion single_stream_reg;
```

Comment:
Maybe we should make this a runtime option. For example, maybe we can add a member function MLMG::setNoGpuSync to control whether we want to avoid gpu sync if possible (and use single stream for safety).

Reply:
hmm NoSyncRegion is a bit difficult to use with a runtime parameter